src/resources/gameModel.js

```diff
@@ -78,4 +78,18 @@ export const getCount = (() => {
   })
 
   return count
+})
+
+export const addGameDoc = ((roomName, roomType, timerID) => {
+  return db.collection("games").add({
```

Comment:
kung pede please change db.collection('games') to gamesCollection and import from firebase.js